lib/actions/adapterActions.js

```diff
@@ -1112,7 +1112,13 @@ export function selectedSerialPort(port) {
     return dispatch => (
         new Promise(() => {
             dispatch(closeAdapter(() => {
-                getDeviceInfo(port.serialNumber)
+                getProbeInfo(port.serialNumber)
+                    .then(probeInfo => {
+                        logger.info('Segger serial: ', probeInfo.serialNumber);
+                        logger.info('Segger speed: ', probeInfo.clockSpeedkHz, ' kHz');
```

Comment:
I'd argue that segger clock speed is not relevant to show in the log view, rather show in log file only (logger.debug)